app/utils/auth.py

```diff
@@ -14,11 +14,13 @@ def access_secret_file(secret_id, version_id="latest"):
 
 # Function to ensure incoming request is from controller with key
 def key_check(api_key: str = Header(None)):
-  
+  print("api key",api_key)
+  print("env type",os.environ['ENV_TYPE'])
```

Comment:
Using print statements to display environment variables, such as `print("env type", os.environ['ENV_TYPE'])`, is not considered a best practice. Printing sensitive information like environment types can lead to security risks and clutter the output with unnecessary details. Instead, consider using proper logging frameworks or avoid printing such values unless absolutely necessary.